GitExtUtils/Delimiters.cs

```diff
@@ -0,0 +1,29 @@
+using System.IO;
+
+namespace System
+{
+    /// <summary>
+    /// Singleton instances of commonly used character arrays.
+    /// </summary>
+    /// <remarks>
+    /// Using these instances avoids allocating an array for each invocation of methods
+    /// such as <c>string.Split</c>.
+    /// </remarks>
+    internal static class Delimiters
+    {
+        public static readonly char[] LineFeed = { '\n' };
+        public static readonly char[] Space = { ' ' };
+        public static readonly char[] Tab = { '\t' };
+        public static readonly char[] Null = { '\0' };
+        public static readonly char[] TabAndSpace = { '\t', ' ' };
+        public static readonly char[] TabAndLineFeedAndCarriageReturn = { '\t', '\n', '\r' };
```

Comment:
Personally I'm finding this less readable than a shorter `TabAndCrLf`. Both `Cr` and `Lf` are established acronyms and both are present in git vocabulary as well.